config/settings.yml

```diff
@@ -36,4 +36,9 @@ acceptable_bcp47_codes:
   - fa-Arab
   - fa-Latn
   - fr
+  - ms
   - tr
```

Comment:
```suggestion
```

Reply:
Removed. I didn't want to remove it if it caused issues with existing data, but it's gone :-)